openpype/hosts/nuke/startup/custom_write_node.py

```diff
@@ -53,24 +58,74 @@
 }
 
 
+class WriteNodeKnobSettingPanel(nukescripts.PythonPanel):
+    """ Write Node's Knobs Settings Panel """
+    def __init__(self):
+        nukescripts.PythonPanel.__init__(self, "Set Knobs Value(Write Node)")
+
+        knobs_value, _ = self.get_node_knobs_setting()
+        # create knobs
+
+        self.typeKnob = nuke.Enumeration_Knob(
+            'override_subsets', 'override subsets', knobs_value)
+        # add knobs to panel
+        self.addKnob(self.typeKnob)
+
+    def process(self):
+        """ Process the panel values. """
+        write_selected_nodes = [
+            s for s in nuke.selectedNodes() if s.Class() == "Write"]
+
+        node_knobs = self.typeKnob.value()
+        ext = None
+        knobs = knobs_setting["knobs"]
+
+        if node_knobs:
+            _, node_knobs_settings = self.get_node_knobs_setting(node_knobs)
+            if not node_knobs_settings:
+                nuke.message("No knobs value found in subset group..\nDefault setting will be used..")  # noqa
+            else:
+                knobs = node_knobs_settings
+
+        for knob in knobs:
+            if knob["name"] == "file_type":
```

Comment:
My guess is that we should require `file_type`, because without it it will fail. We should let know to user if this is failing due to the missing `file_type` knob in settings. Use `nuke.message` please.